src/_pytest/_code/code.py

```diff
@@ -1009,30 +1018,23 @@ def __init__(self, tblines: Sequence[str]) -> None:
         self.extraline = None
 
 
+@attr.s
 class ReprEntryNative(TerminalRepr):
+    lines = attr.ib(type=Sequence[str])
     style = "native"  # type: _TracebackStyle
 
-    def __init__(self, tblines: Sequence[str]) -> None:
-        self.lines = tblines
-
     def toterminal(self, tw: TerminalWriter) -> None:
         tw.write("".join(self.lines))
 
 
+@attr.s
 class ReprEntry(TerminalRepr):
-    def __init__(
-        self,
-        lines: Sequence[str],
-        reprfuncargs: Optional["ReprFuncArgs"],
-        reprlocals: Optional["ReprLocals"],
-        filelocrepr: Optional["ReprFileLocation"],
-        style: "_TracebackStyle",
-    ) -> None:
-        self.lines = lines
-        self.reprfuncargs = reprfuncargs
-        self.reprlocals = reprlocals
-        self.reprfileloc = filelocrepr
-        self.style = style
+    lines = attr.ib(type=Sequence[str])
+    reprfuncargs = attr.ib(type=Optional["ReprFuncArgs"])
+    reprlocals = attr.ib(type=Optional["ReprLocals"])
+    # XXX/NOTE: changed arg name: filelocrepr => reprfileloc
+    reprfileloc = attr.ib(type=Optional["ReprFileLocation"])
```

Comment:
I assume we can go with `s/filelocrepr/reprfileloc`, or should it support the old name?  (it was at least used internally).

Reply:
> I assume we can go with `s/filelocrepr/reprfileloc`, or should it support the old name? (it was at least used internally).

Still an open question..

Reply:
I think it is OK to change the parameter name and keep the attribute `reprfileloc`: I doubt people create `ReprEntry` explicitly (it is not possible to import `ReprEntry` through the public API).